actors/runtime/src/test_utils.rs

```diff
@@ -831,17 +831,17 @@ impl Runtime<MemoryBlockstore> for MockRuntime {
         {
             let mut balance = self.balance.borrow_mut();
             if value > *balance {
-                return Err(actor_error!(SysErrSenderStateInvalid;
-                        "cannot send value: {:?} exceeds balance: {:?}",
-                        value, *balance
+                return Err(ActorError::unchecked(
+                    ExitCode::SYS_SENDER_STATE_INVALID,
+                    format!("cannot send value: {:?} exceeds balance: {:?}", value, *balance),
```

Comment:
The mock has to replicate what the FVM would do sometimes. I have a hunch that ActorError is not a good type to return from the Runtime API.

Actually this code might be wrong, and it should be an "internal send failed", masking the system code.

Reply:
See https://github.com/filecoin-project/builtin-actors/pull/197 for a possible fix